helpers/constants.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
I'd be more inclined to keep these where they were, or to put them on the object/class they relate to, then to create a single constants file like this.

Reply:
moved to their individual modules https://github.com/lbryio/spee.ch/commit/f0435be0b27d25333fbc74939f204b3bd94a444f